extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -6,9 +6,11 @@
 import * as vscode from 'vscode';
 import * as path from 'path';
 import * as os from 'os';
-import { promises as fs, existsSync } from 'fs';
+import * as constants from './../common/constants';
+import * as fs from 'fs';
```

Comment:
the other files in this extension do "import * as promises from 'fs'"

Reply:
yeah because fs.exists() is not a promise thats why, the files that only need promises can do that